src/SyntaxHighlighter/Cpp.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "SyntaxHighlighter/Cpp.h"
+
+#include <QColor>
+#include <QRegularExpression>
+#include <QString>
+#include <QTextCharFormat>
+
+namespace orbit_syntax_highlighter {
+namespace C {
+namespace {
```

Comment:
There might be a reason that I don't see, but why this extra namespace here?

Reply:
I wanted to have the C-namespace because after I want to include the C++ rules and Henning asked for the anonymous also. @beckerhe ??

Reply:
`namespace C` for the name. And the unnamed `namespace` to get internal linkage.

Reply:
There is no way to shorten that. At least none that I know of.

Reply:
Got it, thank you